test/endtoend/operator/101_initial_cluster_vtorc_vtadmin.yaml

```diff
@@ -71,15 +71,6 @@ spec:
   - name: commerce
     durabilityPolicy: semi_sync
     turndownPolicy: Immediate
-    vitessOrchestrator:
-      resources:
-        limits:
-          memory: 128Mi
-        requests:
-          cpu: 100m
-          memory: 128Mi
-      extraFlags:
-        recovery-period-block-duration: 5s
```

Comment:
I have removed this to verify that we still start a VTOrc instance irrespective of whether it is part of the config or not.